<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pc</class>
 <widget class="QDialog" name="pc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>377</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>537</width>
    <height>377</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>537</width>
    <height>377</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>修改个人信息</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>91</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;密码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>141</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;确认密码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>191</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;电子邮件&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>241</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;电话&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="pc_email">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="pc_phone">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pc_regb">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>290</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>确定修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pc_clearb">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>重新输入</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pc_logkey">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="pc_logkeysure">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>只能是数字或字母的组合</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="time">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/mainwindow/pc.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pc_exitb">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/mainwindow/20080528160815502.gif);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/mainwindow/lms.ico);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>pc_email</zorder>
  <zorder>pc_phone</zorder>
  <zorder>pc_regb</zorder>
  <zorder>pc_clearb</zorder>
  <zorder>pc_logkey</zorder>
  <zorder>pc_logkeysure</zorder>
  <zorder>label_6</zorder>
  <zorder>label</zorder>
  <zorder>time</zorder>
 </widget>
 <tabstops>
  <tabstop>pc_logkey</tabstop>
  <tabstop>pc_logkeysure</tabstop>
  <tabstop>pc_email</tabstop>
  <tabstop>pc_phone</tabstop>
  <tabstop>pc_clearb</tabstop>
  <tabstop>pc_regb</tabstop>
  <tabstop>pc_exitb</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
